css/style.css

```diff
@@ -1,3 +1,26 @@
-h1 {
-  color: salmon;
+
+
+body {
+  background:#EDEFEC;
+  text-align: center;
+  /*display:flex;
+  flex-direction: column;
+  justify-content: center;*/
+  /*margin:0 auto;*/
+}
+.content {
+  max-width: 700px;
+  margin:0 auto;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.imagebox1, .imagebox2 {
```

Comment:
I think you could get away with just a single class `.imagebox` that gets applied to each of your flex-container divs, instead of splitting it off in two `.imagebox1, .imagebox2`.